core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/container/postgres/PostgresConfigService.java

```diff
@@ -84,25 +82,39 @@ public void decorateServicePillarWithPostgresIfNeeded(Map<String, SaltPillarProp
 
     private SSLSaltConfig getSslSaltConfigWhenRotateRequired(StackDto stackDto) {
         SSLSaltConfig sslSaltConfig = new SSLSaltConfig();
-        sslSaltConfig.setRootCertsBundle(stackDto.getCluster().getDbSslRootCertBundle());
-        sslSaltConfig.setSslEnabled(stackDto.getCluster().getDbSslEnabled());
+        ClusterView cluster = stackDto.getCluster();
+        String dbSslRootCertBundle = cluster.getDbSslRootCertBundle();
+        // As explained in DatabaseSslDetails, Cluster.dbSslEnabled == null is only possible for legacy clusters, and its effective value may be safely assumed
+        // false. In practice, however, no null value should be ever encountered at this point; it should be already initialized properly by the
+        // DatabaseSslService.getDbSslDetailsForRotationAndUpdateInCluster() invocation from ClusterDbCertRotationHandler.
```

Comment:
I think the log message with the warn message is enough and handling the mentioned edge case, so would you mind remove the commented explanation?